eva_cttv_pipeline/consequence_type.py

```diff
@@ -174,36 +173,12 @@ class ConsequenceType:
     with relationship to ensembl gene IDs and SO terms
     """
 
-    def __init__(self, ensembl_gene_ids=None, so_names=None):
-        if ensembl_gene_ids:
-            self.ensembl_gene_ids = set(ensembl_gene_ids)
-        else:
-            self.ensembl_gene_ids = set()
-        self._ensembl_transcript_id = None
-
-        if so_names is not None:
-            self.so_terms = set([SoTerm(so_name) for so_name in so_names])
-        else:
-            self.so_terms = set()
+    def __init__(self, ensembl_gene_id, so_term):
+        self.ensembl_gene_id = ensembl_gene_id
+        self.so_term = SoTerm(so_term)
```

Comment:
It would probably make more sense to provide an `SoTerm` object as constructor argument.